be/src/exprs/array_functions.tpp

```diff
@@ -1203,4 +1204,75 @@ private:
     }
 };
 
+// Todo:support datatime/data
+template <LogicalType Type>
+class ArrayGenerate {
```

Comment:
ConstColumn is not processed?

Reply:
support ConstColumn like array_generate(1,9,1), but only for interger type which is needed by some customers right now. I will support datatime/data type like select array_generate('2021-01-01','2021-01-05') later